pkg/api/octopus/client.go

```diff
@@ -1,84 +1,107 @@
 package octopus
 
 import (
-	"context"
-	"time"
-
 	"github.com/kyma-incubator/octopus/pkg/apis"
 	oct "github.com/kyma-incubator/octopus/pkg/apis/testing/v1alpha1"
-	"k8s.io/apimachinery/pkg/types"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/serializer"
 	"k8s.io/client-go/kubernetes/scheme"
-	"sigs.k8s.io/controller-runtime/pkg/client"
-	k8sRestClient "sigs.k8s.io/controller-runtime/pkg/client"
-	"sigs.k8s.io/controller-runtime/pkg/client/config"
+	"k8s.io/client-go/rest"
 )
 
+const NamespaceForTests = "kyma-system"
+
 type OctopusInterface interface {
-	ListTestDefinitions() (*oct.TestDefinitionList, error)
-	ListTestSuites() (*oct.ClusterTestSuiteList, error)
-	CreateTestSuite(cts *oct.ClusterTestSuite) error
-	DeleteTestSuite(cts *oct.ClusterTestSuite) error
-	GetTestSuiteByName(name string) (*oct.ClusterTestSuite, error)
+	ListTestDefinitions(opts metav1.ListOptions) (result *oct.TestDefinitionList, err error)
+	ListTestSuites(opts metav1.ListOptions) (result *oct.ClusterTestSuiteList, err error)
+	CreateTestSuite(cts *oct.ClusterTestSuite) (result *oct.ClusterTestSuite, err error)
```

Comment:
Is there a reason returning created test suite?

Reply:
To be consistent with the standard API.